README.md

```diff
@@ -64,11 +50,33 @@ The next file is `~/.gu/security-hq.local.conf`, which should be given by one of
 
 Another file you need will need (here again ask for a team member) is `~/.gu/security-hq-service-account-cert.json`.
 
+### Adding additional AWS accounts for local development
+
+As well as the `security-test` and `security` accounts, you can add additional AWS accounts to your local security-hq. This may be helpful for debugging purposes for example. You will need janus credentials for the account you want to add.
+
+It's really easy to add a new AWS account! Go to `~/.gu/security-hq.local.conf`, which you should have from a colleague. Add a new object to the `AWS_ACCOUNTS` list, like this Deploy Tools account example:
+
+```
+AWS_ACCOUNTS = [
+  {
+   name = "Deploy Tools"
+   id = "deployTools" 
+   roleArn = ""
+  }
+]
+```
+
+The value of `id` should be the same as the AWS Profile name, which you can see when you copy your credentials from janus. You can add a `roleArn` if you want to generate an iAM report, otherwise you don't need it.
```

Comment:
Couple of capitalisation fixes 😸 

```suggestion
The value of `id` should be the same as the AWS Profile name, which you can see when you copy your credentials from Janus. You can add a `roleArn` if you want to generate an IAM report, otherwise you don't need it.
```